validator/walk.go

```diff
@@ -7,8 +7,15 @@ import (
 	"github.com/vektah/gqlparser"
 )
 
+func newEvents() *Events {
+	return &Events{
```

Comment:
Events is meant to be a stateless collection of callbacks, this doesn't belong here. Maybe on Walker?